extras/envoy-xds/main.go

```diff
@@ -27,15 +25,18 @@ import (
 )
 
 var (
-	configPath   string
-	listenAddr   string
-	pollInterval time.Duration
+	configPath          string
+	listenAddr          string
+	pollInterval        time.Duration
+	discoveryClient     *pkg.DiscoveryClient
+	discoveryServerAddr string
 )
 
 func init() {
 	flag.StringVar(&configPath, "config", "", "Path to the config JSON file with Envoy xDS configuration")
 	flag.StringVar(&listenAddr, "listenAddr", "", "Address (host:port) to bind for Envoy xDS")
 	flag.DurationVar(&pollInterval, "pollInterval", 5*time.Second, "Polling interval for checking config updates")
+	flag.StringVar(&discoveryServerAddr, "discoveryServerAddr", "", "Addresses (multi:///host1:port,host2:port,host3:port) of discovery server")
```

Comment:
Could this be also part of the `json` config file instead?

Reply:
done